activerecord/lib/active_record/relation/delegation.rb

```diff
@@ -60,15 +60,24 @@ def generate_method(method)
           return if method_defined?(method)
 
           if /\A[a-zA-Z_]\w*[!?]?\z/.match?(method)
-            module_eval <<-RUBY, __FILE__, __LINE__ + 1
-              def #{method}(*args, &block)
-                scoping { klass.#{method}(*args, &block) }
-              end
-            RUBY
+            if RUBY_VERSION >= "2.7"
+              module_eval <<-RUBY, __FILE__, __LINE__ + 1
+                def #{method}(...)
```

Comment:
Do you prefer to use `...` with `RUBY_VERSION >= "2.7"` version checking?
Actually the warning in this generated method could be addressed by just one `ruby2_keywords(method) if respond_to?(:ruby2_keywords, true)`.
https://github.com/rails/rails/commit/2da55798ef4772977f25b739ab02567c8ded5c64#diff-563ce9140aa399b5ba97bb2a64b7c568R73

Reply:
> Do you prefer to use ... with RUBY_VERSION >= "2.7" version checking?

My reasoning is that `ruby2_keywords` is just a temporary thing to ease the transition, but `...` is here to stay. So I'd rather use it when possible.

I'm fine to use ruby2_keywords instead, if you think it's better.

Reply:
How about `definition = RUBY_VERSION >= "2.7" ? "..." : "*args, &block"` to avoid code duplication?

Reply:
Done.